master/pkg/model/defaults.go

```diff
@@ -97,13 +103,27 @@ func DefaultExperimentConfig() ExperimentConfig {
 		SchedulingUnit:  100,
 		Environment: Environment{
 			Image: RuntimeItem{
-				CPU: "determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-aaa3750",
-				GPU: "determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-aaa3750",
+				CPU: defaultCPUImage,
+				GPU: defaultGPUImage,
 			},
 		},
 		Reproducibility: ReproducibilityConfig{
 			ExperimentSeed: uint32(time.Now().Unix()),
 		},
 		MaxRestarts: 5,
 	}
+
+	if taskContainerDefaults == nil {
+		return defaultConfig
+	}
+
+	defaultConfig.Environment.RegistryAuth = taskContainerDefaults.RegistryAuth
+	defaultConfig.Environment.ForcePullImage = taskContainerDefaults.ForcePullImage
+
+	if taskContainerDefaults.Image != nil {
+		defaultConfig.Environment.Image.CPU = taskContainerDefaults.Image.CPU
+		defaultConfig.Environment.Image.GPU = taskContainerDefaults.Image.GPU
```

Comment:
Can't you just do `defaultConfig.Environment.Image = taskContainerDefaults.Image`?

Reply:
good idea